include/tvm/ir/op.h

```diff
@@ -146,6 +146,7 @@ class OpNode : public RelayExprNode {
   // Internal function to compute if it is primitive op
   bool IsPrimitiveOp_() const {
     const auto& fn_ty = this->op_type;
+    if (fn_ty.get() == 0) return false;
```

Comment:
I am not too sure about the change. We should not use `0` in the case of nullptr.

Ideally all Op should have a Type. So instead, we might want to do CHECK(fn_ty != nullptr) here and make sure we register a type for every Op

Reply:
Thanks for pointing out.

I think I was trying to hash an Op with Type undefined, which led me to `return false` here. Otherwise, I would crash into a segfault error.

This is updated accordingly as suggested.